ruby/cucumber-cucumber-expressions.gemspec

```diff
@@ -23,14 +23,14 @@ Gem::Specification.new do |s|
     'source_code_uri' => 'https://github.com/cucumber/cucumber-expressions/tree/main/ruby',
   }
 
-  s.add_runtime_dependency 'bigdecimal'
+  s.add_dependency 'bigdecimal'
```

Comment:
@luke-hill this looks breaking. Is that correct?

Reply:
Nah this is just minor naming. Although I wasn't aware this was a new recommendation :man_shrugging: